lib/log.js

```diff
@@ -175,105 +169,16 @@ Lug.prototype.notifyAttachedServices = function (name, request, data) {
 // These events indicate key points at which a particular
 // user has interacted with the service.
 
-Lug.prototype.activityEvent = function (event, request, data) {
-  if (! data || ! data.uid) {
-    this.error({ op: 'log.activityEvent', data: data })
-    return
-  }
-
-  var info = {
-    event: event
-  }
-
-  if (request.headers['user-agent']) {
-    info.userAgent = request.headers['user-agent']
-  }
-
-  optionallySetService(info, request)
-
-  Object.keys(data).forEach(function (key) {
-    info[key] = data[key]
-  })
-
-  this.logger.info('activityEvent', info)
-  this.statsd.write(info)
+Lug.prototype.activityEvent = function (data) {
+  this.logger.info('activityEvent', data)
```

Comment:
Noticing that the validation of `data` is moved into the metrics module, which is probably fine, and it's no fun to pay validation costs twice, but that does mean that there is a possibility of us goofing up when using `log.activityEvent` or `log.flowEvent` directly, right?

Not sure if that's a bad thing, but just wanted to pick at it to make sure we thought about it.